mros2_msgs/action/ControlQos.action

```diff
@@ -0,0 +1,5 @@
+#goal definition
+QoS qos
+---
```

Comment:
Is this the action going to the mros_reasoner? I see another one in Jonatan's fork https://github.com/jginesclavero/mc_mros_reasoner/blob/foxy_devel/mc_mros_reasoner_msgs/action/MonitorObjective.action which I think may be better.

We even need to add more information, specifically in the feedback, we should have a string with the selected mode, and in the request.

Also, it may be useful to have an optional "desired" mode, so the metacontroller sets that mode initially @chcorbato what do you think about that?

In any case, I think we should have a couple more information here.

Reply:
Hi @marioney

Check out the new commit with your request. I am packaging all the metacontrol information in the `QoS` message. Maybe we could rename it, but limit all the changes in the messages to this msg would help to be adopted by future users. It is the same message as before using our metacontroller, but with a new `QoS` field.

Best

Reply:
This seems fine by me.  We can use the `QoS` message, the information there seems to be enough.